### ISSUE TITLE
Full linux support

### ISSUE THREAD

**Issue by geloizi** [nabla-containers/nabla-containers.github.io#3]
Sorry for opening an issue. Feel free to resolve it immediately. BTW, do you have plans to start a mailing list?

Is there an possibility to run "full linux" in nabla container? I found this image: https://hub.docker.com/r/nablact/ubuntu/ but it doesn't start.

**Comment by lumjjb** [comment]
No worries, this is a very valid and interesting issue to discuss. Which takes into account various aspects, including trade-offs of security (i.e. potentially exposing more interfaces to support more), the direction of the community (i.e. development of other solo5 unikernels based on different interfaces - LKL, mirage, etc.), as well as the use case of generality (i.e. fork vs event loop). 

In terms of direct linux specific support. There are several approaches can think about: adding a compatibility layer between rump and the linux interface, or just running LKL directly on top of solo5. @djwillia and @ricarkol probably can shed more light on the developments in this area.

Currently, we maintain a list of known limitations that we've listed which we are tackling: https://github.com/nabla-containers/runnc#limitations

>BTW, do you have plans to start a mailing list?

That is a good suggestion, we will discuss this and decide what we should use. Slack is a potential candidate too! 

>Is there an possibility to run "full linux" in nabla container? I found this image: https://hub.docker.com/r/nablact/ubuntu/ but it doesn't start.

That's probably a debugging container we used.  : ). Sorry if it caused any confusion.
